cluster/gce/gci/configure-helper-common.sh

```diff
@@ -617,9 +626,7 @@ function create-master-auth {
     for extra_component in "${extra_components[@]}"; do
       local token="$(secure_random 32)"
       append_or_replace_prefixed_line "${known_tokens_csv}" "${token}," "system:${extra_component},uid:system:${extra_component}"
-      if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-        create-kubeconfig "${extra_component}" "${token}" "${PROXY_RESERVED_IP}" "8888" "http"
-      elif [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
+      if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
if ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE ?

Reply:
will address along with the decision 1) name of the variable to controller the insecure/secure port  2) two variables: one for enable insecure port, one for use insecure ports in kubemarktest

Reply:
fixed as commented above for the two new flags in system and removal of ENABLE_APISERVER_INSECURE_PORT.